csharp/Worker/Microsoft.Spark.CSharp/Worker.cs

```diff
@@ -343,12 +343,20 @@ private static IFormatter ProcessCommand(Stream inputStream, Stream outputStream
                         continue;
                     }
 
-                    WriteOutput(outputStream, serializerMode, message, formatter);
+                    try
+                    {
+                        WriteOutput(outputStream, serializerMode, message, formatter);
+                    }
+                    catch (Exception)
+                    {
+                        logger.LogError("WriteOutput() failed at iteration {0}", count);
+                        throw;
+                    }
                     count++;
                     funcProcessWatch.Start();
                 }
 
-                logger.LogDebug("Output entries count: " + count);
+                logger.LogInfo("Output entries count: " + count);
```

Comment:
this "info" give us a sense of total entries be processed by a worker, just like the "info" for "func process time". Based on the "info" of "Output entries count" and "func process time", we can have the idea of processing time of each process.
So I still think it is better to use "info" than "Debug".